src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -111,10 +111,21 @@ public static class CLIOptions
             ArgumentShortName = "-compare",
             ArgumentDescription = $@"Enables comparing to results stored in Stryker Dashboard. This feature is only available in combination with {Diff.ArgumentName}",
             ValueType = CommandOptionType.NoValue,
-            JsonKey = "dashboard-diff-compare"
+            JsonKey = "dashboard-compare"
 
         };
 
+
+        public static readonly CLIOption<string> BaselineStorageLocation = new CLIOption<string>
+        {
+            ArgumentName = "--baseline-storage-location",
+            ArgumentShortName = "-bsl <storageLocation>",
+            ArgumentDescription = $@"Allows to choose a storage location, this can be either azure, disk or dashboard. When the dashboard reporter is enabled, the dashboard will be the default storage location. Otherwise disk is the default storage location. Selecting azure allows you to store baselines using your own instance of azure file storage.",
```

Comment:
The azure file storage option is actually called azurefilestorage, not azure. Perhaps it would be better to use an Enum instead of hard coded strings to avoid mistakes like these.